packages/migrate/migrations/routes/index.js

```diff
@@ -59,7 +59,7 @@ export async function migrate() {
 		config.kit?.routes ??
 		((filepath) => !/(?:(?:^_|\/_)|(?:^\.|\/\.)(?!well-known))/.test(filepath));
 
-	const files = glob(`${routes}/**`, { filesOnly: true });
+	const files = glob(`${routes}/**`, { filesOnly: true }).map((file) => file.replace(/\\/g, '/'));
```

Comment:
oh huh — i thought it would always return paths with `/` regardless of platform. i may have confused input with output https://github.com/terkelg/tiny-glob#windows